### DISCUSSION TITLE
添加悬浮球显示在左侧的选项

### DISCUSSION THREAD

**Discussion by SammySnake-d** [immersive-translate/immersive-translate#2268] [Ideas]
### 新功能 | New Feature

添加悬浮球显示在左侧的选项

### 为什么您想要添加这些功能 | Why you want to add these features

1.有些页面是内嵌的滑动组件，例如chatgpt，悬浮球在右侧会遮挡部分滑动组件
2.某些插件也是在页面右侧显示，而左侧显示的插件较少，右侧页面会造成插件布局重叠的情况
3.

**Comment by keyboard3** [comment]
![QQ_1731320508520](https://github.com/user-attachments/assets/dd789fd9-d658-44c7-8470-655c481f5fd9)
